components/Sprite.jsx

```diff
@@ -3,7 +3,14 @@ import styled from 'styled-components';
 import gsap from "gsap";
 
 
-const Sprite = ({ spriteClickHandler, isSpriteInSpeechBox, spriteClicked, isInitialSpriteClick, isSpriteNavButton}) => {
+const Sprite = ({ spriteClickHandler, isSpriteInSpeechBox, spriteClicked, isInitialSpriteClick, isSpriteNavButton, location}) => {
```

Comment:
These props aren't giving me a great sense of what they're doing. 
spriteClicked vs isInitialSpriteClick is a distinction that I'm sure is important, but I can't grasp it here.
As far as I know, we have a click handler for when the sprite is clicked, why do we need a state that says the sprite is clicked? Can the sprite be unclicked? What would that look like?